prompts.json

```diff
@@ -0,0 +1,26 @@
+[
+{"title_prompt": "For all my years, I never knew you could _"},
+{"title_prompt": "If i had a nickel for _, I would be rich"},
+{"title_prompt": "When I die, I want _"},
+{"title_prompt": "Growing up, I had an attraction to _"},
+{"title_prompt": "How I was able to _"},
+{"title_prompt": "My story on how I got stuck in  _"},
+{"title_prompt": "How the lord gave me life for me to give you _"},
+{"title_prompt": "Doctors hate me because _"},
+{"title_prompt": "I should’ve went to school instead of _"},
+{"title_prompt": "Why you do not simply _"},
+{"title_prompt": "1"},
```

Comment:
if the title_prompt 1-14 are filler, can we add an underscore next to each number so they can be used in testing when we develop the game

(if there's no underscore and the frontend is expecting an input it might break so just to be safe lets add an _)